rfcfold

```diff
@@ -334,10 +334,10 @@ process_input() {
     warn "'-c' option ignored when unfolding (option '-r')"
   fi
 
-  if [[ "$strategy" -eq 2 ]]; then
-    min_supported=$(expr ${#hdr_txt_2} + 9)
+  if [[ "$strategy" -eq 0 ]] || [[ "$strategy" -eq 2 ]]; then
+    min_supported=$(expr ${#hdr_txt_2} + 8)
   else
-    min_supported=$(expr ${#hdr_txt_1} + 9)
+    min_supported=$(expr ${#hdr_txt_1} + 8)
   fi
   if [[ "$maxcol" -lt "$min_supported" ]]; then
     err "the folding column cannot be less than $min_supported."
```

Comment:
I'm unsure about this.  I played before with '8' vs '9'.  8 is the exact number, but since the message on line 343 says "not less than", I bumped it to 9.   I was going to change the text to say "not less than or equal to", but that created a line that was too long, causing `rfcfold` to itself be folded when put into the draft.   

I'll check this after merging...

Reply:
I checked manually that the output is correct with 4 leading `=`, the header, and 4 trailing `=`.

The error message is emitted afer a _less than_ check (`-lt`), thus it is correct to complain about a value less than the minimum.